### PR TITLE
New package: IridiumIO.CompactGUI version 3.2.1

### PR DESCRIPTION
**Pull request by ksharizard**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/144079)

### CODE REVIEW

**Comment by GhoulBoii** [comment]
@microsoft-github-policy-service agree

**Comment by stephengillie** [comment]
Automatic Validation ended with:


(Automated response - build 870.)

**Comment by stephengillie** [comment]
image

This appears to be `Microsoft.DotNet.DesktopRuntime.6`.

File: manifests/i/IridiumIO/CompactGUI/3.2.1/IridiumIO.CompactGUI.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: IridiumIO.CompactGUI
+PackageVersion: 3.2.1
+InstallerType: exe
+Installers:
```
Comment (stephengillie) [inline manifests/i/IridiumIO/CompactGUI/3.2.1/IridiumIO.CompactGUI.installer.yaml:7]:
```suggestion
Dependencies:
  PackageDependencies:
   - PackageIdentifier: Microsoft.DotNet.DesktopRuntime.6
Installers:
```

(Automated response - build 870.)

File: manifests/i/IridiumIO/CompactGUI/3.2.1/IridiumIO.CompactGUI.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: IridiumIO.CompactGUI
+PackageVersion: 3.2.1
+InstallerType: exe
```
Comment (stephengillie) [inline manifests/i/IridiumIO/CompactGUI/3.2.1/IridiumIO.CompactGUI.installer.yaml:6]:
```suggestion
InstallerType: portable
```

Hi @GhoulBoii,

Sorry for so many separate updates. This appears to be a `portable` type package.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**